Improve docs (#681)
festim-dev/FESTIM: docs/source/userguide/materials.rst

> Materials are key components of hydrogen transport simulations. They hold the properties like diffusivity, solubility and even thermal properties like thermal conductivity or heat capacity.

Reviewer: ```suggestion
Materials are vital components of hydrogen transport simulations. They hold diffusivity, solubility and thermal properties like thermal conductivity or heat capacity.
```

Parameters description
----------------------

The :class:`festim.Material` class has three required arguments:

> * :code:`id`: a unique id given to the material/volume. It is useful when defining volumetric source terms or exports. Several ids can be given to the same material if multiple volumes have the same material.

Reviewer: ```suggestion
* :code:`id`: a unique id given to the material/volume. It is useful when defining volumetric source terms or exports. Several id's can be given to the same material if multiple volumes have the same material.
```

* :code:`thermal_cond`: the thermal conductivity in W/m/K
* :code:`heat_capacity`: the heat capacity in J/kg/K
* :code:`rho`: the volumetric density in kg/m3
* :code:`H`: the heat of transport in J/mol that takes a dictionary {"free_enthalpy": …, "entropy": …} so that H = free_enthalpy + entropy*T. For more information see :ref:`Soret effect`.

--------------------
Integration with HTM
--------------------

> H-transport-materials (HTM) is a python database of hydrogen transport properties.

Reviewer: ```suggestion
H-transport-materials (HTM) is a Python database of hydrogen transport properties.
```
